conda_forge_tick/new_update_versions.py

```diff
@@ -30,7 +30,10 @@
 
 
 def get_latest_version(
-    name: str, payload_meta_yaml: Any, sources: Iterable[AbstractSource],
+    name: str,
+    payload_meta_yaml: Any,
+    sources: Iterable[AbstractSource],
+    versions: dict,
```

Comment:
you might consider having this return a dict rather than mutating one.

Reply:
This way I can attach the `bad` string when it occurs, but I will try another approach